src/quill-cursors/quill-cursors.ts

```diff
@@ -110,6 +111,15 @@ export default class QuillCursors {
   private _registerDomListeners(): void {
     const editor = this.quill.container.getElementsByClassName('ql-editor')[0];
     editor.addEventListener('scroll', () => this.update());
+    editor.addEventListener('touchstart', this._handleCursorTouch);
+  }
+
+  private _handleCursorTouch(e: MouseEvent): void {
+    const touchFlagDelay = 2000;
```

Comment:
Shouldn't we use `this._hideDelay` here?

Reply:
Yeah, changed it to `this.options.hideDelayMs`